### ISSUE TITLE
OpenFiles not opening files when bookdown project created in RStudio

### ISSUE THREAD

**Issue by ismayc** [rstudio/bookdown#1094]
I noticed in https://github.com/rstudio/bookdown/blob/master/inst/rstudio/templates/project/skeleton.dcf that `README.md`, `_bookdown.yml`, `_output.yml`, and `index.Rmd` are supposed to open when a {bookdown} project is created in RStudio, but they aren't for me. [Here](https://www.loom.com/share/aa6fbb4b0b3f4e479de99d3ded0cf7dc) is a Loom video of what I see.

```{r}
> xfun::session_info('bookdown')
R version 4.0.3 (2020-10-10)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Catalina 10.15.7, RStudio 1.4.461

Locale: en_US.UTF-8 / en_US.UTF-8 / en_US.UTF-8 / C / en_US.UTF-8 / en_US.UTF-8

Package version:
  base64enc_0.1.3   bookdown_0.21     digest_0.6.27     evaluate_0.14     glue_1.4.2       
  graphics_4.0.3    grDevices_4.0.3   highr_0.8         htmltools_0.5.1.1 jsonlite_1.7.2   
  knitr_1.31        magrittr_2.0.1    markdown_1.1      methods_4.0.3     mime_0.10        
  rlang_0.4.10      rmarkdown_2.7     stats_4.0.3       stringi_1.5.3     stringr_1.4.0    
  tinytex_0.30      tools_4.0.3       utils_4.0.3       xfun_0.21         yaml_2.2.1  
```

---

By filing an issue to this repo, I promise that

- [x] I have fully read the issue guide at https://yihui.org/issue/.
- [x] I have provided the necessary information about my issue.
    - If I'm asking a question, I have already asked it on Stack Overflow or RStudio Community, waited for at least 24 hours, and included a link to my question there.
    - If I'm filing a bug report, I have included a minimal, self-contained, and reproducible example, and have also included `xfun::session_info('bookdown')`. I have upgraded all my packages to their latest versions (e.g., R, RStudio, and R packages), and also tried the development version: `remotes::install_github('rstudio/bookdown')`.
    - If I have posted the same issue elsewhere, I have also mentioned it in this issue.
- [x] I have learned the Github Markdown syntax, and formatted my issue correctly.

I understand that my issue may be closed if I don't fulfill my promises.

Note that I did not try `remotes::install_github('rstudio/bookdown')` here since it's been a few years since the `skeleton.dcf` file for {bookdown} was edited. Hope that's OK and happy to do so if that fixes things of course.

Thanks!

**Comment by cderv** [comment]
Hi, 

Thanks for the report.

I try on my end and I can't reproduce. Here is also a [Loom video](https://www.loom.com/share/f7f620e4e9264eb9b0359c044cd79496) to show what I have. 

As it is rather a RStudio IDE issue, I then looked into their issue board and found the culprit: https://github.com/rstudio/rstudio/issues/7940

It seems it has been fixed in RStudio 1.4. Which version are you using ?

**Comment by cderv** [comment]
I think you should upgrade to released 1.4 version (which is now 1.4.1106) in which I think it has been fixed
https://rstudio.com/products/rstudio/download/#download

**Comment by ismayc** [comment]
I can confirm that it now works after upgrading to 1.4.1106. Thanks!
